src/main/java/appeng/integration/modules/theoneprobe/tile/CraftingMonitorInfoProvider.java

```diff
@@ -48,7 +48,8 @@ public void addProbeInfo( AEBaseTile tile, ProbeMode mode, IProbeInfo probeInfo,
 
 			if( displayStack != null )
 			{
-				final ItemStack itemStack = displayStack.getItemStack();
+				// TODO: check if OK
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1135)